apps/randomcats/randomcats.go

```diff
@@ -0,0 +1,25 @@
+// Package randomcats provides details for the Random Cats applet.
+package randomcats
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed random_cats.star
+var source []byte
+
+// New creates a new instance of the Random Cats applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "random-cats",
+		Name:        "Random Cats",
+		Author:      "mrrobot245",
+		Summary:     "Shows pictures of cats",
+		Desc:        "Shows random pictures of cats/gifs of cats.",
```

Comment:
Would be great to give a call out to cataas.com as well. It's such a wonderful service they provide! =)

Reply:
Added!